pkg/service/service.go

```diff
@@ -24,41 +24,40 @@ const dockerConfigDir = "/var/lib/faasd/.docker/"
 // Remove removes a container
 func Remove(ctx context.Context, client *containerd.Client, name string) error {
 
-	container, containerErr := client.LoadContainer(ctx, name)
-
-	if containerErr == nil {
-		found := true
-		t, err := container.Task(ctx, nil)
-		if err != nil {
-			if errdefs.IsNotFound(err) {
-				found = false
-			} else {
-				return fmt.Errorf("unable to get task %s: ", err)
-			}
+	container, err := client.LoadContainer(ctx, name)
+	if err != nil {
+		service := client.SnapshotService("")
+		key := name + "snapshot"
+		if _, err := client.SnapshotService("").Stat(ctx, key); err == nil {
+			service.Remove(ctx, key)
 		}
+	}
 
-		if found {
-			status, _ := t.Status(ctx)
-			fmt.Printf("Status of %s is: %s\n", name, status.Status)
-
-			log.Printf("Need to kill %s\n", name)
-			err := killTask(ctx, t)
-			if err != nil {
-				return fmt.Errorf("error killing task %s, %s, %s", container.ID(), name, err)
-			}
+	found := true
+	t, err := container.Task(ctx, nil)
+	if err != nil {
+		if errdefs.IsNotFound(err) {
+			found = false
+		} else {
+			return fmt.Errorf("unable to get task %s: ", err)
 		}
+	}
 
-		err = container.Delete(ctx, containerd.WithSnapshotCleanup)
+	if found {
+		status, _ := t.Status(ctx)
+		fmt.Printf("Status of %s is: %s\n", name, status.Status)
```

Comment:
is this right? `fmt.Printf` should it be `log` or just deleted?

Reply:
I think it's preexisting.

Reply:
but it should be a log statement right?

Reply:
I think you might be right.